ui.frontend/src/view/FormField.js

```diff
@@ -77,10 +77,14 @@ class FormField {
     setActive() {
         if (!this.isActive()) {
             this.element.setAttribute(Constants.DATA_ATTRIBUTE_ACTIVE, true);
```

Comment:
this code should be moved to handler which listens to any change in active state.

Reply:
when activeChild is changed a Model Change event is triggered that notifies its dependants and that's how the setActive gets triggered in view. We don't have any event in view that handles it currently. Should I create one?